app/assets/stylesheets/modules/_cya-tax-summary.scss

```diff
@@ -13,6 +13,17 @@
 
 }
 
+div.tai-tax-summary-table:nth-last-of-type(1) dl {
```

Comment:
Not sure about this part.Rest all looks good

Reply:
its basically a hack  to make it look like the prototype - remember yesterday in retro I mentioned the prototype had a table but staging we use a summary list component and we kind of ended up with a hybrid solution where we were formatting the summary list so it looks like the table in the prototype.  The use of a Table in the prototype is incorrect in my opinion